test/integration/CustomsMetricsConnectorSpec.scala

```diff
@@ -40,15 +40,24 @@ class CustomsMetricsConnectorSpec extends IntegrationTestSpec with GuiceOneAppPe
   private implicit val vpr: ValidatedPayloadRequest[AnyContentAsXml] = TestData.TestCspValidatedPayloadRequest
   private implicit val mockExportsLogger: ExportsLogger = mock[ExportsLogger]
 
-  override protected def beforeAll() {
+  /**
+   * On Jenkins the localhost string is different to when run locally.
+   *
+   * @return
+   */
+  def localhostString(): String = {
```

Comment:
Seems like a code smell that we have to do this to test log messages :(

Reply:
yeah is a weird thing def needs refactoring at some point